config/api.yml

```diff
@@ -2315,6 +2316,35 @@
       :post:
       - :name: remove
         :identifier: svc_catalog_provision
+  :cloud_templates:
+    :description: Cloud Templates
+    :identifier: template
+    :options:
+      - :collection
+      - :subcollection
+    :verbs: *g
+    :klass: ManageIQ::Providers::CloudManager::Template
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: image_show_list
+      :post:
+      - :name: query
+        :identifier: image_show_list
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: image_show
+      - :name: query
```

Comment:
query should only be a collection action

Reply:
@jntullo Ah, sorry missed it.
Fixed.